internal/controllers/clusterrequest_controller.go

```diff
@@ -705,6 +716,88 @@ func (t *clusterRequestReconcilerTask) handleClusterInstallation(ctx context.Con
 	return nil
 }
 
+// handleClusterPolicyConfiguration updates the ClusterRequest status to reflect the status
+// of the policies that match the managed cluster created through the ClusterRequest.
+func (t *clusterRequestReconcilerTask) handleClusterPolicyConfiguration(ctx context.Context) error {
+	if t.object.Status.ClusterInstanceRef == nil {
+		return fmt.Errorf("status.clusterInstanceRef is empty")
+	}
+	// Get all the child policies in the namespace of the managed cluster created through
+	// the ClusterRequest.
+	policies := &policiesv1.PolicyList{}
+	listOpts := []client.ListOption{
```

Comment:
What about scoping to client.InNamespace(t.object.Status.ClusterInstanceRef.Name) instead of all namespaces?